packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -148,6 +153,11 @@ def get_configuration_and_output_metadata_by_index(
 
     output_parameter_list = []
     for output in metadata.measurement_signature.outputs:
+        if output.message_type and output.message_type != "ni.protobuf.types.DoubleXYData":
+            raise click.ClickException(
+                "Measurement outputs with message datatype other than DoubleXYData are not supported."
```

Comment:
Include `output.message_type` similar to this:
f"Measurement outputs do not support {output.message_type}. Only DoubleXYData is supported."